hack/dashboard/openshift/deploy-grafana.sh

```diff
@@ -28,6 +28,24 @@ validate_cluster() {
 	header "Validating cluster"
 
 	local -i ret=0
+
+	local oc_version=""
+	oc_version=$(oc version --client -oyaml | grep releaseClientVersion | cut -f2 -d:)
+
+	local -i oc_major_version oc_minor_version
+	oc_major_version=$(echo "$oc_version" | cut -f1 -d.)
+	oc_minor_version=$(echo "$oc_version" | cut -f2 -d.)
+
+	[[ $oc_major_version -eq 4 ]] && [[ $oc_minor_version -ge 12 ]] || {
+		fail "oc version $oc_version used should be at least 4.12.0"
+		info "install a newer version of oc"
+		cat <<-EOF
+			     curl -sNL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.13.0/openshift-client-linux.tar.gz |
+				   tar -xzf - -C .
+		EOF
+		ret=1
```

Comment:
will the script continue from here onwards?

Reply:
yes, but the return value will be 1 so that script function will continue with all validations and then exit
```
   🔆🔆🔆  Validating cluster  🔆🔆🔆
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Error from server (NotFound): namespaces "openshift-monitoring" not found
 ❌ FAIL: openshift-monitoring namespace missing. Is this an OpenShift cluster?
 🔔 cluster:  https://127.0.0.1:40853
 ❌ FAIL: Missing Kepler CRD. Is Kepler Operator Installed?
error: the server doesn't have a resource type "kepler"
 ❌ FAIL: Mising kepler deployment. Did you create kepler instance?
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ❌ FAIL: Cluster validation failed!
 🔔 Fix issues reported above and rerun the script
```